app/src/main/java/org/oppia/android/app/walkthrough/topiclist/WalkthroughTopicViewModel.kt

```diff
@@ -31,6 +32,24 @@ class WalkthroughTopicViewModel @Inject constructor(
     Transformations.map(topicListSummaryResultLiveData, ::processTopicListResult)
   }
 
+  val _isHeaderTextViewVisible =  MutableLiveData<Boolean>()
```

Comment:
As we know the layout is in the activity, right. 
And our codebase is in the `WalkthroughTopicListFragmentPresenter` which is basically the fragment side. So, try to make it like a fragment is talking to the activity by passing the info. 

Take example of `TopicLessonsFragmentPresenter and TopicActivity`

Reply:
Thanks, fixed this